lms/views/admin/application_instance.py

```diff
@@ -33,32 +33,30 @@ def instances(self):  # pylint: disable=no-self-use
         request_method="POST",
         renderer="lms:templates/admin/instance.new.html.jinja2",
     )
-    def instance_new(self):
-        lti_registration_id = self.request.matchdict["id_"]
+    def new_instance(self):
+        lti_registration = self.lti_registration_service.get_by_id(
+            self.request.matchdict["id_"]
+        )
```

Comment:
It's a little confusing that the `id_` field to new application instance is the id of a registration?